components/blocks/gridLayout.tsx

```diff
@@ -23,13 +33,32 @@ export const GridLayout = ({ data }: GridLayoutProps) => {
     <>
       {data.grids?.map((grid, i) => (
         <Container padding="pt-0" key={i}>
-          <h2 className="text-xl text-ssw-red">{grid.gridTitle}</h2>
+          {grid.showGridTitle && (
+            <h2
+              data-tina-field={tinaField(grid, "gridTitle")}
+              className={`${grid.centeredGridTitle ? "text-center text-2xl font-light" : "text-xl text-ssw-red"} my-3`}
+            >
+              {grid.gridTitle}
+            </h2>
+          )}
```

Comment:
You can probably remove `showGridTitle` from the schema 
```suggestion
          {grid.gridTitle && (
            <h2
              data-tina-field={tinaField(grid, "gridTitle")}
              className={`${grid.centeredGridTitle ? "text-center text-2xl font-light" : "text-xl text-ssw-red"} my-3`}
            >
              {grid.gridTitle}
            </h2>
          )}
```

Reply:
As discussed, im using the bool to allow for showing label in Tina.

Pushed change adding in check for gridTitle before rendering the header